lib/matplotlib/transforms.py

```diff
@@ -2975,3 +2976,26 @@ def offset_copy(trans, fig=None, x=0.0, y=0.0, units='inches'):
         y /= 72.0
     # Default units are 'inches'
     return trans + ScaledTranslation(x, y, fig.dpi_scale_trans)
+
+
+class ScaledRotation(Affine2DBase):
```

Comment:
Please place this near `ScaledTranslation`, and preferably add a few tests (though I see that `ScaledTranslation` doesn't have any, so testing this one indirectly via the polar stuff is okay for now too.)

Also, as I'm not sure how generic this transform is, I'm not sure if it should be public; any preferences @timhoffm?

Reply:
No clear preference on public. I believe this is not useful for anybody else. OTOH, transforms are quite technical anyway and I don't think API will need to change, so no harm in making it public. Overall, maybe just make it private, so that somebody reading on Transforms is bothered with it (it's not so easy to understand what this is).

Reply:
Alright, let's make it private for now, then.